build/nlu-regression/current_scores/bpds-intent.json

```diff
@@ -35,49 +35,49 @@
       "metric": "accuracy",
       "problem": "bpsd A",
       "seed": 69,
-      "score": 0.6818181818181818
+      "score": 0.6863636363636364
```

Comment:
Can we round this at 3 decimals, would be easier to consult :)

Reply:
I only round it when I log it in console:

![image](https://user-images.githubusercontent.com/25970722/100364715-7c4a2580-2fcc-11eb-98c7-9f631a55e0be.png)

these files are not meant to be consulted directly, but they actually help me know if something has changed.

I'm more confident if results are the exact same with 12 digits